deploy-ecs/deploy.sh

```diff
@@ -25,7 +25,7 @@ function check_deployment_complete() {
   deployment_details="${1}"
 
   # get and print current task counts
-  desired_count="$(echo "${deployment_details}" | jq -r '.desiredCount')"
+  desired_count="$(echo "${deployment_details}" | jq -r '[.desiredCount, 1] | max')"
```

Comment:
So this assumes that `desired_count` should always be at least `1`? That could cause spurious failures if we have a service's task count intentionally set to `0`, but that's an edge case.

Reply:
If you desire 0 tasks, what is being deployed?

Reply:
- A different container image tag
- Changes from the template task definition

Sometimes we have to take a service down to perform maintenance and then bring it back up with a different version / different configuration that accounts for the maintenance. Not a big deal, just something we'll have to keep in mind in that situation.